Makefile

```diff
@@ -236,9 +240,10 @@ endif
 ifeq ($(llvm_version),3.7.1)
 else ifeq ($(llvm_version),3.8.1)
 else ifeq ($(llvm_version),3.9.1)
+else ifeq ($(llvm_version),4.0.0)
 else
   $(warning WARNING: Unsupported LLVM version: $(llvm_version))
-  $(warning Please use LLVM 3.7.1, 3.8.1, or 3.9.1)
+  $(warning Please use LLVM 3.7.1, 3.8.1, 3.9.1, or 4.0.0)
```

Comment:
i thought we were only "supporting" the most recent version of any of the major releases. if we are supporting 4.0.0 then we need to do CI on it. Homebrew is going to install 4.0.1 on OSX.